custom_components/tuya_local/devices/poolsana_heatpump.yaml

```diff
@@ -0,0 +1,113 @@
+name: PoolSana InverPower Next
+primary_entity:
+  entity: climate
+  translation_key: pool_heatpump
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: Cooling
+              value: cool
+            - dps_val: Heating
+              value: heat
+            - dps_val: Auto
+              value: auto
+    - id: 2
+      type: string
+      name: mode
+      hidden: true
+    - id: 4
+      type: integer
+      name: temperature
+      range:
+        min: 70
+        max: 400
+      mapping:
+        - scale: 10
+          step: 10
+    - id: 16
+      type: integer
+      name: current_temperature
+      range:
+        min: -300
+        max: 1000
+      mapping:
+        - scale: 10
+          step: 1
+    - id: 5
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: Boost
+          value: boost
+        - dps_val: Silence
+          value: eco
+        - dps_val: Smart
+          value: comfort
+secondary_entities:
+  - entity: sensor
+    class: temperature
+    name: Effluent Temperature
+    category: diagnostic
+    dps:
+      - id: 25
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 100
+        unit: "°C"
+        mapping:
+          - scale: 1
+            step: 1
+  - entity: sensor
```

Comment:
There is no need to duplicate the current_temperature and temperature from the climate entity.

Reply:
Tile card and Entity card both let you display the current temperature or temperature of a climate entity by itself easily, if that is what you want to do.  I think the entities list card you have to edit the yaml to do it, and maybe some custom cards do not support it, but in that case you can add a template_sensor helper to expose the attribute as a sensor if you really want to.